tests/test_celery.py

```diff
@@ -0,0 +1,124 @@
+import pytest
+from django.apps import apps
+from django.conf import settings
+from mock import patch
+
+from health_check.contrib.celery.apps import HealthCheckConfig
+from health_check.contrib.celery.backends import CeleryHealthCheck
+
+
+class TestCeleryHealthCheck:
+    HEALTH_CHECK_CLASS = CeleryHealthCheck
+    CELERY_APP_CONTROL_PING = (
+        "health_check.contrib.celery.backends.app.control.ping"
+    )
+    CELERY_APP_CONTROL_INSPECT_ACTIVE_QUEUES = (
+        "health_check.contrib.celery.backends.app.control.inspect.active_queues"
+    )
+
+    def test_check_status_doesnt_add_errors_when_ping_successfull(self):
+        with patch(
+            self.CELERY_APP_CONTROL_PING,
+            return_value=[{"celery@4cc150a7b49b": {"ok": "pong"}}],
+        ), patch(
+            self.CELERY_APP_CONTROL_INSPECT_ACTIVE_QUEUES,
+            return_value={
+                "celery@4cc150a7b49b": [
+                    {"name": queue.name} for queue in settings.CELERY_QUEUES
+                ]
+            },
+        ):
+            health_check = self.HEALTH_CHECK_CLASS()
+            health_check.check_status()
+
+            assert len(health_check.errors) == 0
```

Comment:
NIP: following one is also valid and more pythonic:

```suggestion
            assert not health_check.errors
```